### PR TITLE
add FCVAR_REPLICATED to ns_server_name

### PR DESCRIPTION
**Pull request by catornot**
Tested, works correctly. Although it's not getting reset after leaving a server (idk how to fix this).

### CODE REVIEW

**Comment by F1F7Y** [comment]
> Although it's not getting reset after leaving a server (idk how to fix this).

That's exactly how FCVAR_REPLICATED works though.

**Comment by catornot** [comment]
I am making proximity chat mod which is supposed to 100% server side. to differentiate servers apart I need some id, In this case its the server's name. if `ns_server_name` doesn't have `FCVAR_REPLICATED` flag the clients wouldn't know the name of the server. 

this would be useful for this [pr](https://github.com/R2Northstar/NorthstarMods/pull/536) in Northstar mods

**Comment by BobTheBob9** [comment]
> Can you gimme a quick ELI5 why we want that?
> 
> (My convar knowledge is minimal and giving a reason for the change helps me better understand it ^^)

just lets clients know the name of the server they're currently on

**Review by BobTheBob9 — CHANGES_REQUESTED** [review]
not much reason not to replicate description as well

**Comment by catornot** [comment]
now `ns_server_desc` as FCVAR_REPLICATED too

**Review by BobTheBob9 — APPROVED** [review]
